videos/nappn_2021/walkthrough.Rmd

```diff
@@ -0,0 +1,340 @@
+- 9:00 - 9:25: Using trait data with R
+- 9:25 - 9:30: break
+- 9:30 - 9:50: Using weather data with R, combining trait & weather data
+- 9:50 - 10:00: TERRA REF data review, wrap-up
+
+### Download/install
+
+1. Sign up for Globus account on [https://www.globus.org/](https://www.globus.org/)
+2. Download Globus Connect Personal
+    - At [https://www.globus.org/globus-connect-personal](https://www.globus.org/globus-connect-personal)
+    - Follow automated setup steps
+3. Download R and RStudio
+    - R at [https://www.r-project.org/](https://www.r-project.org/)
+    - RStudio at [https://rstudio.com/products/rstudio/download/](https://rstudio.com/products/rstudio/download/)
+    - R packages: raster, dplyr, ggplot2
+4. Download Dryad data
+    - Download manually at [https://datadryad.org/stash/dataset/doi:10.5061/dryad.4b8gtht99](https://datadryad.org/stash/dataset/doi:10.5061/dryad.4b8gtht99) by clicking blue "Download dataset" button OR
+    - Download with R using code below
+
+```{r, eval=FALSE}
+local_path <- "dryad_data.zip"
+download.file(url = "https://datadryad.org/stash/downloads/file_stream/624637", destfile = local_path)
+unzip(local_path)
+```
+
+## Data overview
+
+The purpose of this walkthrough is to provide an introduction the TERRA REF public domain high throughput phenomics datasets (LeBauer et al 2020), and to give learners a chance to access and explore a representative cross section of these data. 
+
+In addition to this workshop, we have online tutorials for using these data at [https://terraref.github.io/tutorials/](https://terraref.github.io/tutorials/). There are also materials from previous walkthrough workshops at [https://github.com/terraref/tutorials/tree/master/videos](https://github.com/terraref/tutorials/tree/master/videos). We are striving to make these data as accessible and usable as possible. 
+
+TERRA REF is a project funded by the Advanced Research Projects Agency for Energy (ARPA-E). The objective is to advance the science and technology of high throughput field phenotyping, which is the automated measurement of plants. TERRA REF has deployed an exceptionally large robot capable of collecting data from a range of sensors at 1 mm spatial resolution every other day. You can learn more about the project from the website, terraref.org. The suite of sensors include visible light cameras, hyperspectral cameras, a laser 3d scanner, and environmental data. In addition there are weather stations and high resolution genome sequences for hundreds of varieties of Sorghum being monitored. 
+
+### Experimental design
+
+(Show picture of site) Field site is in Arizona. Field is passed over by large robot called a gantry. Lot of equipment in hanging box to collect many types of data. 
+
+Gantry goes systematically over entire field once a day. Some sensors take data every day, like 9,000 images from RGB camera. Others more intermittently, like hyperspectral images, because of data space limits. 
```

Comment:
```suggestion
The field scanner systematically scans the entire field, typically every other day during the growing season, but in some cases multiple times per day.
```

This is a good place for the figure of sensor data products x time from the README

Reply:
I present that figure during the **Dryad data** section later on, when introducing the README, but should it go here instead/also?

Reply:
should be fine down there